src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You are linking a SCSS file in your HTML. However, browsers can't understand SCSS, they only understand CSS. You need to compile your SCSS into CSS and link the CSS file here instead.